Apps/W1/HybridGP/app/src/Migration/Customers/GPCustomerMigrator.codeunit.al

```diff
@@ -173,10 +173,18 @@ codeunit 4018 "GP Customer Migrator"
             CopyStr(MigrationGPCustomer.ADDRESS2, 1, 50), Country, CopyStr(MigrationGPCustomer.ZIPCODE, 1, 20),
             CopyStr(MigrationGPCustomer.CITY, 1, 30));
         CustomerDataMigrationFacade.SetContact(CopyStr(MigrationGPCustomer.CNTCPRSN, 1, 50));
+
+        if (CopyStr(MigrationGPCustomer.PHONE1, 1, 14) = '00000000000000') then
```

Comment:
Minor - Consider introducing a function to check if it is blank and protentional generating the 00000000000000 or hardcoding it as a string. You could also try Replace(0,'') and that one would make it blank.

Reply:
I have created shared function to handle this.